src/storages/key_value_store.js

```diff
@@ -223,8 +224,8 @@ export class KeyValueStore {
             validator: ow.isValid(k, ow.string.matches(KEY_VALUE_STORE_KEY_REGEX)),
             message: 'The "key" argument must be at most 256 characters long and only contain the following characters: a-zA-Z0-9!-_.\'()',
         })));
-        if (options.contentType && !ow.isValid(value, ow.any(ow.string, ow.buffer))) {
-            throw new ArgumentError('The "value" parameter must be a String or Buffer when "options.contentType" is specified.', this.setValue);
+        if (options.contentType && !(ow.isValid(value, ow.any(ow.string, ow.buffer)) || isStream(value))) {
```

Comment:
let's remove the dependency and simplify the checks, i guess we could just allow objects, in the end the `is-stream` thing does that too and only checks for existence of a method

https://github.com/sindresorhus/is-stream/blob/main/index.js#L1-L5

```suggestion
        if (options.contentType && !(ow.isValid(value, ow.any(ow.string, ow.buffer, ow.object)))) {
```

I'd like to ship new stable in few hours, so if you want to have it there, please hurry :]

Reply:
Alright, let me fix it!